tee-worker/service/src/prometheus_metrics.rs

```diff
@@ -209,6 +219,77 @@ impl ReceiveEnclaveMetrics for EnclaveMetricsReceiver {
 	}
 }
 
+fn handle_callback_calls(call: &TrustedCall) {
+	if let Ok(mut hashmap) = STF_REQUEST_EXT_HASH.lock() {
+		match call {
+			TrustedCall::link_identity(_, _, _, ValidationData::Web2(_), _, _, hash) => {
+				hashmap.insert(*hash, std::time::Instant::now());
+			},
+			TrustedCall::link_identity_callback(_, _, identity, _, hash) => {
+				if let Some(time) = hashmap.remove(hash) {
```

Comment:
I think this will result it stale entries if we run multiworker setup with synced trusted operation pool.
The map will only grow larger and larger because callbacks can be eventually processed by different workers.

Reply:
I think this should be ultimately cleaned up [here](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/sidechain/consensus/aura/src/block_importer.rs#L126) if we want to really measure that. Given that TOPs can be processed by different workers I'm not sure this metric will produce valuable results.

Reply:
But does our current implementation support another worker processing the STF Tasks?

Reply:
No, but I think we should take that into account. What do you think?

Reply:
I don't really have a preference here, but I'd argue we wait for concrete implementation that we can test with multiple workers and have a separate issue for it. We still have to somehow measure how much time it takes for a request to be completed.